lib/platform/linux-i2c.c

```diff
@@ -69,6 +69,11 @@ struct switchtec_i2c {
 	((struct switchtec_i2c *) \
 	 ((char *)d - offsetof(struct switchtec_i2c, dev)))
 
+#define IS_MRPC_CMD_ADDR(x) (((size_t)x & 0xfff) == 0x800)
+#define IS_MRPC_NO_RETRY_CMD(x) (((x & 0xfff) > 0x100) && \
+				 ((x & 0xfff) < 0x120) && \
+				 ((x & 0xfff) != 0x107))
```

Comment:
That is way too gross.

Create an `i2c_gas_write32_noretry()` and a `__gas_write32_noretry()`  which uses it for I2C (and is a regular gas_write32 for other platforms)...  then use it, when appropriate, in gasop_cmd().

Reply:
Updated. Please review.